requirements/adg/base.txt

```diff
@@ -1,3 +1,4 @@
 mailchimp3==3.0.14
 pytest-mock==3.3.1
 mandrill==1.0.59
+-e git+https://github.com/philanthropy-u/edx-msp-assessment-xblock.git#egg=msp-assessment-xblock
```

Comment:
do we have ssh setup on adg server ??

Reply:
@qasimgulzar maybe the devops team can shed light on that. However, if we look at edx requirements file, they have added the xblock in the same way, via https:

From `edx-platform/requirements/edx/base.txt`:
`-e git+https://github.com/edx/DoneXBlock.git@2.0.3#egg=done-xblock`

Reply:
to add in like above you have shared, we will have to make our repo public.

Reply:
not necessarily, if we provide the username and password of a Github account that has access to the repo, it will allow the installation, or am I missing something?